Analysis/source/bbeTauAnalyzer.cxx

```diff
@@ -30,7 +30,7 @@ class bbetauAnalyzer : public BaseEventAnalyzer {
         region.SetCharge(os);
 
         for(auto wp = working_points.rbegin(); wp != working_points.rend(); ++wp) {
-            if(tau->Passed(signalObjectSelector.GetTauVSjetDiscriminator().first, *wp)) {
+            if(tau->Passed(signalObjectSelector.GetTauVSjetDiscriminator().at(ana_setup.mode).first, *wp)) {
```

Comment:
why the signature of `GetTauVSjetDiscriminator` has changed? It should return a pair, not a map!